data-loader/core/src/main/java/com/scalar/db/dataloader/core/util/ColumnUtils.java

```diff
@@ -0,0 +1,74 @@
+package com.scalar.db.dataloader.core.util;
+
+import com.scalar.db.common.error.CoreError;
+import com.scalar.db.dataloader.core.exception.Base64Exception;
+import com.scalar.db.io.BigIntColumn;
+import com.scalar.db.io.BlobColumn;
+import com.scalar.db.io.BooleanColumn;
+import com.scalar.db.io.Column;
+import com.scalar.db.io.DataType;
+import com.scalar.db.io.DoubleColumn;
+import com.scalar.db.io.FloatColumn;
+import com.scalar.db.io.IntColumn;
+import com.scalar.db.io.TextColumn;
+import java.util.Base64;
+
+/** Utility class for dealing and creating ScalarDB Columns */
+public final class ColumnUtils {
+  private ColumnUtils() {
+    // restrict instantiation
+  }
+
+  /**
+   * Create a ScalarDB column from the given data type, column name, and value. Blob source values
+   * need to be base64 encoded.
+   *
+   * @param dataType Data type of the specified column
+   * @param columnName ScalarDB table column name
+   * @param value Value for the ScalarDB column
+   * @return ScalarDB column
+   * @throws Base64Exception if an error occurs while base64 decoding
+   */
+  public static Column<?> createColumnFromValue(DataType dataType, String columnName, String value)
```

Comment:
Adding `＠Nullable` for nullable parameters would be helpful:
```suggestion
  public static Column<?> createColumnFromValue(DataType dataType, String columnName, ＠Nullable String value)
```

Reply:
Added.